builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -615,15 +624,15 @@ function store.update_paths()
 	pkgmgr.update_gamelist()
 	for _, game in pairs(pkgmgr.games) do
 		if game.author ~= "" and game.release > 0 then
-			local id = game.author:lower() .. "/" .. game.id
+			local id = game.author:lower():encode_uri_component() .. "/" .. game.id
```

Comment:
id is used to track packages locally, and should never be used in a URL as case info is lost